### PR TITLE
Adding parameters to smallfile test

### PR DESCRIPTION
**Pull request by Avilir**
Adding parameters to the test of smallfiles, and updating the corresponding yaml file.

### CODE REVIEW

File: tests/e2e/test_small_file_workload.py [hunk 1]
```diff
@@ -51,6 +60,24 @@ def test_smallfile_workload(self, ripsaw):
...
+        sf_data['spec']['workload']['args']['file_size'] = file_size
+        sf_data['spec']['workload']['args']['files'] = files
+        sf_data['spec']['workload']['args']['threads'] = threads
+        sf_data['spec']['workload']['args']['samples'] = samples
+        ''' Calculating the size of the volume that need to be test, it should be at least twice in the size then the
+             size of the files, and at least 100Gi.
+             since the file_size is in Kb and the vol_size need to be in Gb, more calculation is needed.
+        '''
+        vol_size = int(files * threads * file_size * 3)
+        vol_size = int(vol_size / 1024 / 1024)
```
Comment (RazTamir) [inline tests/e2e/test_small_file_workload.py]: please use contants.GB2KB (which is 1024^2)

File: tests/e2e/test_small_file_workload.py [hunk 2]
```diff
@@ -51,6 +60,24 @@ def test_smallfile_workload(self, ripsaw):
 
         log.info("Running SmallFile bench")
         sf_data = templating.load_yaml(constants.SMALLFILE_BENCHMARK_YAML)
+
+        '''
```
Comment (RazTamir) [inline tests/e2e/test_small_file_workload.py]: please use """ (3 X ") instead of ''' (3 X ')

Fix for all inline comments

**Comment by keesturam** [comment]
@Avilir LGTM. please post the test result output.

**Comment by vasukulkarni** [comment]
LGTM, needs jenkins test run.

**Pull request merged**